powershell/tests/functions/Invoke-Maester.Tests.ps1

```diff
@@ -3,5 +3,53 @@
         if (Get-MgContext) { Disconnect-Graph } # Ensure we are disconnected
         { Invoke-Maester } | Should -Throw 'Not connected to Microsoft Graph.*'
     }
-}
 
+    It 'Validates smoke test results' {
+        if (Get-MgContext) { Disconnect-Graph } # Ensure we are disconnected
+
+        $maesterParams = @{
+            Path                 = [System.IO.Path]::GetFullPath((Join-Path -Path (Split-Path -Path $PSScriptRoot -Parent) -ChildPath "smoketests"))
+            OutputFolder         = [System.IO.Path]::GetFullPath((Join-Path -Path (Split-Path -Path $PSScriptRoot -Parent) -ChildPath "test-results"))
+            PassThru             = $true
+            SkipGraphConnect     = $true
+            NonInteractive       = $true
+            OutputFolderFileName = "TestResults"
+            ExcludeTag           = "testtag"
+            NoLogo               = $true
+        }
+        $r = Invoke-Maester @maesterParams
```

Comment:
This is going to seem like a nit-pick, but all variable names should have a clear meaning and purpose.